Code Quality/Reviewing a pull request.md

```diff
@@ -0,0 +1,72 @@
+> Optimism is an occupational hazard of programming; feedback is the treatment. - Kent Beck
+
+This guide should help you review code better. A good code review will provide a lot of useful things, but some of them are:
+
+-   a sense of security (that the feature works well),
+-   a learning opportunity for both parties,
+-   architectural check,
+-   code quality check.
+
+## Language of the code review
+
+-   Be concise
+-   Do not ask, advise
+-   Do not assume the level of knowledge
+-   Be open for discussion, but do not bike shed
+-   Recommend fixes instead of pointing out errors
+
+### Good examples:
+
+> this won't work because the function you're calling is `async`.
+
+> this file is placed in the incorrect folder, it should go to `components/shared`
```

Comment:
for this example I think we need an explanation why would it be better to move that file, like 
```
> this file should go to `components/shared` because this is atom component and can be reused on another page
```

Or something like that. If we give an explanation it might open a discussion about it.

Reply:
Amazing thanks